src/api/app/views/webui/groups/_group_members.html.haml

```diff
@@ -42,9 +42,9 @@
           Add Member
   = render partial: 'add_group_user_modal', locals: { group: group }
 
-  = render DeleteConfirmationDialogComponent.new(modal_id: 'delete-group-members-modal',
-                                                 method: :delete,
-                                                 options: { modal_title: 'Do you want to remove this group member?' })
+= render DeleteConfirmationDialogComponent.new(modal_id: 'delete-group-members-modal',
```

Comment:
This change was needed for the modal to work when the logged-in user isn't a group member or an admin, so `write_access` on line 36 is `false`.